1.0.2 changes

cs-abhishek-shukla opened the pull request:
- Release notes
- Structure and navigation
- Typos and grammar

cs-suyog-jinturkar commented on release_notes.md:
```diff
@@ -1,4 +1,7 @@
-- Previously, the widget's installation would fail because the deployment package of the solution pack only allowed for a specific version of the SOC Management widget. This meant that the installation couldn't accommodate the latest version. However, the dependency on the specific version has been removed, and now the solution pack will always install the most up-to-date version of the SOC Manag ...
\ No newline at end of file
+
+- Removed the dependency on a specific version, and now the solution pack installs the most up-to-date version of the SOC Management widget
```
imo, both convey almost similar points, can we combine them?

cs-suyog-jinturkar commented on docs/usage.md:
```diff
@@ -1,17 +1,21 @@
 
 - Below is an example of the Dashboard with Demo records
 
     ![SOC Management Dashboard](res/soc_mngt_dashboard.png)
```
I feel, it will be very lucrative and exciting for an end user to see the sample dashboard in overview (landing page). Can we move this image to overview section then?

Note: Please make sure that image is uptodate as well

cs-suyog-jinturkar replied:
@cs-abhishek-shukla

cs-abhishek-shukla committed:
1.0.2 changes
- Release notes
- Structure and navigation
- Typos and grammar
- Feedback incorporated
